static/bidder-info/loopme.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: http://prebid.loopmertb.com
```

Comment:
Endpoint is reachable:
curl -i --location --request POST http://prebid.loopmertb.com
HTTP/1.1 204 No Content
date: Thu, 14 Nov 2024 08:44:47 GMT